tools/actions/turbo-affected/build/main.js

```diff
@@ -18595,8 +18595,10 @@ var package_default = {
     "coin:bitcoin": "pnpm --filter coin-bitcoin",
     "coin:evm": "pnpm --filter coin-evm",
     "coin:framework": "pnpm --filter coin-framework",
+    "coin:tester": "pnpm --filter coin-tester",
```

Comment:
This might be unrelated change...

Reply:
You are right. I'll delete it. Thanks for realizing it!